cloud/blockstore/libs/notify/notify.h

```diff
@@ -57,7 +58,7 @@ struct IService
 
 ////////////////////////////////////////////////////////////////////////////////
 
-IServicePtr CreateService(TNotifyConfigPtr config);
+IServicePtr CreateService(TNotifyConfigPtr config, NCloud::NIamClient::IIamTokenClientPtr IamTokenClient);
```

Comment:
Does it work: `NCloud::NIamClient::IIamTokenClientPtr` -> `NIamClient::IIamTokenClientPtr` ?